### PR TITLE
Also allow active elments for non-buttons & show pointer on buttons

### PR DESCRIPTION
**Pull request by MorrisJobke**
* have no `action` specified but want to make the entry active doesn't work before this

### CODE REVIEW

File: src/components/PopoverMenu/PopoverMenuItem.vue [hunk 1]
```diff
@@ -45,7 +45,7 @@
 		</a>
 
 		<!-- If item.input is set instead, an put will be used -->
-		<span v-else-if="item.input" class="menuitem">
+		<span v-else-if="item.input" class="menuitem" :class="{'active':(item.active)}">
```
Comment (skjnldsv) [inline src/components/PopoverMenu/PopoverMenuItem.vue]: `:class="{active: item.active }"` ?

**Comment by MorrisJobke** [comment]
I fixed the comments by @skjnldsv and this is now working fine over here 👍

**Comment by skjnldsv** [comment]
LGTM! :ship:

**Comment by skjnldsv** [comment]
@MorrisJobke merge

**Comment by MorrisJobke** [comment]
released as 0.6.3

**Pull request merged**